data-es7.js

```diff
@@ -78,6 +78,16 @@ exports.browsers = {
   firefox41: {
     full: 'Firefox',
     short: 'FF 41',
+    obsolete: true,
```

Comment:
I would lean towards having browsers in a separate file. Editing the giant es5/es6/es7 file always makes me feel like "oh shit I will break something", and separating them would also sort out the mess of missing versions/not using the latest version/etc which is also a mess right now.